### DISCUSSION TITLE
How do I personalize the theta theme buttons?

### DISCUSSION THREAD

**Discussion by Lucashdml** [goolord/alpha-nvim#279] [Q&A]
this are my configs
  {
    "goolord/alpha-nvim",
    lazy = false,
    dependencies = {
      "nvim-tree/nvim-web-devicons",
      "nvim-lua/plenary.nvim",
    },
    config = function()
      local alpha = require ("alpha")
      local config = require("alpha.themes.theta").config
      -- there's no name, access via index
      config.layout[2].val = {
        [[                                                                       ]],
        [[                                                                       ]],
        [[                                                                       ]],
        [[                                                                       ]],
        [[                                                                     ]],
        [[       ████ ██████           █████      ██                     ]],
        [[      ███████████             █████                             ]],
        [[      █████████ ███████████████████ ███   ███████████   ]],
        [[     █████████  ███    █████████████ █████ ██████████████   ]],
        [[    █████████ ██████████ █████████ █████ █████ ████ █████   ]],
        [[  ███████████ ███    ███ █████████ █████ █████ ████ █████  ]],
        [[ ██████  █████████████████████ ████ █████ █████ ████ ██████ ]],
        [[                                                                       ]],
        [[                                                                       ]],
        [[                                                                       ]],
      }

      alpha.setup(config)
    end,
  },

**Comment by Yair-Ziv** [comment]
Hey did you find out anything? I just came across this same issue, seems like you might need to copy paste the whole theta theme and modify the buttons but I wonder if there's an easier way
